app/tests/test_basic.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+"""Tests for Basic Functions"""
+import sys
+import json
+import unittest
+
+sys.path.append("../..")
```

Comment:
@ParthS007 Any alternative to this?

Reply:
I have tried other ways but I get error in importing the modules but this works fine, so I used this 👍